Page Design Delivered including styling

patrickf949 opened the pull request:
@pbkabali @joegeoffrey @probuse @@oma0256, @BarnaTB, @armstrongsouljah @calvinpete @AtamaZack
Implemented Styling to templates.
Your reviews will be of great help

pbkabali commented:
Nice work.
Responsiveness..on point. Can't see thou the part where admin can view all the records details from all attendants.

patrickf949 commented:
> Nice work.
> Responsiveness..on point. Can't see thou the part where admin can view all the records details from all attendants.

Thanks @pbkabali  attendants records are on the admin dashboard just after the totals, I'm making a page where individual attendant records can be accessed after a click. 
I Will work on that too though

patrickf949 commented:
@pbkabali I have added the missing pages, including the 'all records' page as well

- Improved on the styling

- Added the edit product, and edit attendant pages

- All records page

- Individual attendant records page
